WordPress/src/main/java/org/wordpress/android/ui/accounts/helpers/FetchBlogListWPCom.java

```diff
@@ -44,6 +44,9 @@ protected void fetchBlogList(final Callback callback) {
                     site.put("isAdmin", jsonSite.get("user_can_manage"));
                     site.put("isVisible", jsonSite.get("visible"));
 
+                    // store capabilities as a text blob
+                    site.put("capabilities", jsonSite.get("capabilities").toString());
```

Comment:
Actually, the request never returns the response without `capabilities` and we are not guarding the ones we know it's going to be there as you can see in the above ones. That's why I didn't guard it.